vllm_gaudi/v1/worker/hpu_model_runner.py

```diff
@@ -732,6 +732,13 @@ def __init__(
                                                       scheduler_config=vllm_config.scheduler_config,
                                                       lora_config=vllm_config.lora_config).tokenizer
 
+        if self.vllm_config.parallel_config.data_parallel_size > 1 and htorch.utils.internal.is_lazy(
+        ) and not self.model_config.enforce_eager:
+            from vllm import envs
+            # disable device group for dp synchronization when hpu graph is
+            # turned on since it's not captured and causes issues
+            envs.VLLM_DISABLE_NCCL_FOR_DP_SYNCHRONIZATION = True
```

Comment:
What does the flag actually do in vllm, can we always set it as true to disable nccl?